examples/CampaignManagement/AddCompleteCampaignsUsingMutateJob.cs

```diff
@@ -0,0 +1,579 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Ads.GoogleAds.Lib;
+using Google.Ads.GoogleAds.V3.Common;
+using Google.Ads.GoogleAds.V3.Errors;
+using Google.Ads.GoogleAds.V3.Resources;
+using Google.Ads.GoogleAds.V3.Services;
+using Google.Api.Gax;
+using Google.LongRunning;
+using Google.Protobuf.WellKnownTypes;
+using System;
+using System.Collections.Generic;
+using static Google.Ads.GoogleAds.V3.Enums.AdGroupAdStatusEnum.Types;
+using static Google.Ads.GoogleAds.V3.Enums.AdGroupCriterionStatusEnum.Types;
+using static Google.Ads.GoogleAds.V3.Enums.AdGroupTypeEnum.Types;
+using static Google.Ads.GoogleAds.V3.Enums.AdvertisingChannelTypeEnum.Types;
+using static Google.Ads.GoogleAds.V3.Enums.BudgetDeliveryMethodEnum.Types;
+using static Google.Ads.GoogleAds.V3.Enums.CampaignStatusEnum.Types;
+using static Google.Ads.GoogleAds.V3.Enums.KeywordMatchTypeEnum.Types;
+using static Google.Ads.GoogleAds.V3.Resources.MutateJob.Types;
+
+namespace Google.Ads.GoogleAds.Examples.V3
+{
+    /// <summary>
+    /// This code example adds complete campaigns including campaign budgets, campaigns, ad groups
+    /// and keywords using MutateJobService.
+    /// </summary>
+    public class AddCompleteCampaignsUsingMutateJob : ExampleBase
+    {
+        /// <summary>
+        /// The number of campaigns to add.
+        /// </summary>
+        private const int NUMBER_OF_CAMPAIGNS_TO_ADD = 2;
+
+        /// <summary>
+        /// The number of ad groups to add.
+        /// </summary>
+        private const int NUMBER_OF_AD_GROUPS_TO_ADD = 2;
+
+        /// <summary>
+        /// The number of keywords to add.
+        /// </summary>
+        private const int NUMBER_OF_KEYWORDS_TO_ADD = 5;
```

Comment:
The value should be 4 as per @BenRKarl feedback

Reply:
_Done_